src/js/views/components/header.ts

```diff
@@ -24,6 +25,7 @@ export class Header implements Widget {
             this.searchForm = new SearchForm();
             this.searchForm.sigSearch.connect((query: string) => {
                 this.sigSearch.emit(query);
+                this.sigLoaderLP.emit();
```

Comment:
it makes no sense to be sending two signals. it should be sufficient to listen to sigSearch.

also - i can't seem to find the place where you'd listen for sigLoaderLP, so it's probably obsolete anyways.